docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -249,11 +249,11 @@ However, accessing the user portfolio page is a little tricky. In development, G
 Simply clicking the **"Preview Custom 404 Page"** button will forward you to the correct page.
 
 <details>
-   <summary>
-      How to sign in when working locally (screenshot)
-   </summary>
-   <br>
-   <img src="https://user-images.githubusercontent.com/29990697/71541249-f63cdf00-2923-11ea-8a85-cefb6f9c9977.gif" alt="How to sign in when working locally">
```

Comment:
same problem with `<details>`

Reply:
Also renders.

Reply:
I don't know why
but I'm just seeing the code itself
may be my editor problem 😅